.github/issue_template.md

```diff
@@ -1,7 +1,36 @@
-## Hello! 
+<!-- IMPORTANT: add the `iroha2` for IROHA2-related issues -->
```

Comment:
note to add `iroha2` label?

Reply:
Yep. I personally forget to add the label.